build.gradle.kts

```diff
@@ -24,7 +24,7 @@ containerBuild {
     version = "1.0.0"
 
     // Project Root Package
-    projectPackage = "org.veupathdb.service.demo"
+    projectPackage = "org.veupathdb.service.sequence"
```

Comment:
Wojtek, i think the package name should be org.veupathdb.service.sequenceRetrieval

Reply:
Hmmm... don't really like camel case in package names, and prefer them as short as possible.  What about simply "srt"?

Reply:
Changed "sequence" to "sr" - "srt" is already taken by the sequence retrieval tool, I think "new SRT" and "old SRT" would be confusing hence this is a service for sequence retrieval, in short service.sr